lib/utils/Queue.ts

```diff
@@ -48,6 +47,7 @@ export class Queue<T> {
     throw Error("Empty queue");
   }
 
+  //???Should the head and tail be reset???
```

Comment:
Do you want to commit this?

Reply:
It is just a reminder for a potential improvement/correction. I have since moved the comment to another branch.